src/main/java/core/basesyntax/service/impl/OperationHandlerBalance.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.impl;
```

Comment:
Please extract handlers to separate package or move to strategy package in order to make service package not so overwhelmed